server/routerlicious/packages/services/src/redisThrottleStorageManager.ts

```diff
@@ -31,6 +32,7 @@ export class RedisThrottleStorageManager implements IThrottleStorageManager {
 
         client.on("error", (error) => {
             winston.error("Throttle Manager Redis Error:", error);
+            Lumberjack.error("Throttle Manager Redis Error:", error);
```

Comment:
Also, can you confirm with @znewton if we want to send Redis "throttling" data to the throttling table, or if we want to keep it in the default table? For throttling, we would have to include this Lumberjack property:

```
{
                [CommonProperties.telemetryGroupName]: "throttling",
}
```

Reply:
Ah yes another good catch. Correct. We'd want this in throttling. It'll make it easier to associate redis failures with Throttling issues